### PR TITLE
Feature flags

### PR DESCRIPTION
**Pull request by Alec8888**
Feature flag to disable the creation of gender restricted rides. 

I think just stopping creating of gender restricted rides is enough to disable the feature. Gender restricted rides that already exist should not be affected by the feature flag, meaning rides that are restricted should not become unrestricted if the flag is disabled.

### CODE REVIEW

File: featureFlags.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const featureFlags = {
+    genderRestrictedRides: true,
```
[the remark this answers is not in the document]
Comment (Alec8888) [reply featureFlags.ts:2]: Yeah that makes sense I think. So featureFlags.ts would have states for both prod and dev and we only env is different between dev and prod?

[the remark this answers is not in the document]
Comment (bakermo) [reply featureFlags.ts:2]: Yes exactly.

[the remark this answers is not in the document]
Comment (bakermo) [reply featureFlags.ts:2]: Like feature flags can still be an object but just return a different object based on whether it is production or development.

**Comment by Alec8888** [comment]
Closing, completing on new PR.

**Pull request closed without merging**